lib/components/primitive-components/Trace/Trace.ts

```diff
@@ -596,6 +596,157 @@ export class Trace
       }
     }
   }
+  __doInitialSchematicTraceRenderWithNetLabelSymbol(
```

Comment:
```suggestion
  _doInitialSchematicTraceRenderWithDownwardSymbol(
```

Reply:
- We don't want to do this for all schematic symbols, just "downward symbols" like ground, we still want the old handling for other symbols
- convention is `_` not `__` (minor)